Brokerages/Oanda/OandaRestApiV1.cs

```diff
@@ -134,74 +134,72 @@ public override bool PlaceOrder(Order order)
                 { "instrument", SymbolMapper.GetBrokerageSymbol(order.Symbol) },
```

Comment:
`OandaRestApiV1`, is deprecated since May 2018 and don't think the endpoints are even working any more:
https://developer.oanda.com/rest-live/introduction/

Maybe we could remove the class completely?

Reply:
Yes your right, I'll verify if we are not using it, if so, I'll remove it 👍

Reply:
I've requested Oanda to confirm that the v1 REST API isn't used for any case/users. I think for this PR I'll leave it, I suspect there are changes in the cloud required too

Reply:
we can remove V1 support following confirmation from Oanda and in a separate PR